### DISCUSSION TITLE
HTML tags with the Office namespace are not removed

### DISCUSSION THREAD

**Discussion by rokka** [mysticmind/reversemarkdown-net#337] [General]
HTML Tags like `<o:p>` are not removed by default.

Is this espected behaviour or is there an option to remove them?

Greetings
Andre

**Comment by mysticmind** [comment]
With regards to ReverseMarkdown config, `UnknownTagsOption.PassThrough` is the default for handling `UnknownTags`. `<o:p>` will be rendered in content _as is_ which is the expected behaviour.

The following are the options available for handling Unknown html tags.
- `UnknownTagsOption.PassThrough` - Include the unknown tag completely into the result. That is, the tag along with the text will be left in output. This is the default.
- `UnknownTagsOption.Drop` - Drop the unknown tag and its content.
- `UnknownTagsOption.Bypass` - Ignore the unknown tag but try to convert its content.
- `UnknownTagsOption.Raise` - Raise an error to let you know.

You could look at using `UnknownTagsOption.Drop`.

All of the above are provided as docs in the [readme](https://github.com/mysticmind/reversemarkdown-net)
